scripts/export/v0_33_0/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	app "github.com/cosmos/cosmos-sdk/cmd/gaia/app"
+	"github.com/cosmos/cosmos-sdk/scripts/export"
+)
+
+const chainID = "cosmoshub-2"
+
+// Command: go run main.go --path=<path_to_old_genesis.json> --start-time=<genesis-start-time> > [path_to_new_genesis.json
+func main() {
+	cdc := app.MakeCodec()
+
+	pathToGenesis := flag.String("path", "", "./genesis.json")
+	genesisTime := flag.String("start-time", "", "2019-02-11T12:00:00Z")
+
+	flag.Parse()
+
+	err := export.ValidateBasic(*pathToGenesis, *genesisTime)
+	if err != nil {
+		panic(err)
+	}
+
+	genesis, err := export.NewGenesisFile(cdc, *pathToGenesis)
+	if err != nil {
+		panic(err)
+	}
+
+	genesis.ChainID = chainID
+	genesis.GenesisTime = *genesisTime
+
+	// proposal #1 updates
+	genesis.AppState.MintData.Params.BlocksPerYear = 4855015
+
+	// proposal #2 updates
+	genesis.ConsensusParams.Block.MaxGas = 200000
```

Comment:
I believe these values are reversed from the values in the proposal [here](https://ipfs.ink/e/Qmam1PU39qmLBzKv3eYA3kMmSJdgR6nursGwWVjnmovpSy)

(max bytes should be 200,000 and max gas 2,000,000)

Reply:
oh thanks for noticing that